native.js

```diff
@@ -46,7 +46,20 @@ If this is important to you, please consider supporting Rollup to make a native
 
 const packageBase = imported.musl && isMusl() ? imported.musl : imported.base;
 const localName = `./rollup.${packageBase}.node`;
-const { parse, parseAsync, xxhashBase64Url } = require(
+const requireWithFriendlyError = id => {
+	try {
+		return require(id);
+	} catch (error) {
+		throw new Error(
+			`Cannot find module ${id}. ` +
+				`npm has a bug related to optional dependencies (https://github.com/npm/cli/issues/4828). ` +
+				'If you are/were using them, please try `npm i` again after removing both package-lock.json and node_modules directory.',
```

Comment:
> If you are/were using them

If a user ended up in this file, they were definitely using them as this Rollup build relies on them. So maybe rather

```suggestion
				'Please try `npm i` again after removing both package-lock.json and node_modules directory.',
```

Reply:
I meant "If you are/were using npm". I forgot to change "them" to "it" after changing "npm@8 and npm@9" to "npm". But I guess it's clear without the if part.